.prettierrc

```diff
@@ -1,2 +1,7 @@
-printWidth: 80
```

Comment:
Can we keep printWidth? it makes Markdown look nicer.

Reply:
I think 80 is the default for Prettier. I don't think this line is strictly needed.

Reply:
Great, we'll run with it as then.